web/src/app/components/dashboard/dashboard.component.ts

```diff
@@ -31,6 +39,15 @@ export class DashboardComponent implements OnInit {
       });
   }
 
+  renderUserDashboardDetails() {
+    const provider = this.route.snapshot.paramMap.get('provider');
+    const displayName = this.route.snapshot.paramMap.get('displayName');
+    this.userService.getUserDashboardDetails(provider, displayName)
+      .subscribe((value) => {
+        this.getUserSocialDetails(value.uid);
```

Comment:
Rxjs chain here would be better

Reply:
@eddiejaoude I am just calling out a function inside subscribe. Could you please let me know what exactly I have to change ?

Reply:
The `this.getUserSocialDetails` in the `subscribe` won't be waited for and if it fails it can not be handled. It would be better to put this in as part of the `pipe`

Reply:
okay @eddiejaoude  👍